synthtool/gcp/common.py

```diff
@@ -58,6 +59,11 @@ def _generic_library(self, directory: str, **kwargs) -> Path:
 
         t = templates.TemplateGroup(self._template_root / directory, self.excludes)
 
+        repo = kwargs["metadata"]["repository"]
```

Comment:
This looks reasonable to me, but I think it would be worth adding a test here:

https://github.com/googleapis/synthtool/blob/master/tests/test_templates.py

Reply:
added a test